docs/Features/Morphology/Dilation.md

```diff
@@ -4,6 +4,8 @@ sidebar_position: 20
 
 import DilateDemo from './dilate.demo.tsx'
 
+_Enlarges the size of foreground objects by iteratively expanding their boundaries, adding pixels based on specific criteria._
```

Comment:
What is meant here by "specific criteria"?

Feels misleading because I could be interpreted as "customisable". If the explanation is too specific, you can just remove it from the one liner.